english/turing-test/eliza/ask_eliza.md

```diff
@@ -14,6 +14,6 @@ Your memory skills at retrieving model numbers of arcane cathod-ray tube devices
 in the late seventies and early eighties are rewarded with the
 following enigmatic response from ELIZA:
 
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.